### DISCUSSION TITLE
Stripe liberado no BR, recomendam já usar? Ele tem vantagens?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [frontendbr/forum#1873] [Discussão]
Bom dia galera!

Hoje recebi um e-mail do pessoal da stripe avisando que liberaram a operação no Brasil, agora pdoemos usar ele como gateway de pagamento. Mas ainda estou um pouco por fora das features dele...

Vocês indicam já implementar no e-commerce que dou manutenção? Ele tem vantagens frente a outros gateways? Por quê ele está tão famoso? ouço falar muito dele...

Valeu galera, abraços!

**Comment by tcelestino** [comment]
É preciso entender qual seria o custo tanto de negócio quanto de desenvolvimento para realizar a implementação do Stripe em seu projeto. 

Alguns pontos que ao meu ver você precisaria levar em consideração:

- meu gateway atual está me atendendo ou não
- quais são as vantagens e desvantagens e migrar para o Stripe (sei que você perguntou, mas já fez você mesmo essa pesquisa?)
- o que essa migração vai trazer de benefícios para o cliente final

É meio "perigoso" achar que porque todo mundo fala "bem" ou "mal" de algo novo que pode acontecer com seu negócio.

**Comment by wilsonneto-swe** [comment]
Então, @tcelestino extao, concordo com teus pontos, por isso estou aqui querendo levantar opiniões, e sobre a sua pergunta de se eu já fiz esta pesquisa, bom, é isso que estou fazendo aqui... rs

Sim, já li toda a documentação, já levantei vários pontos, mas aqui não estou atrás de especificações, mas sim de opiniões de quem já tenha usado em projetos internacionaisou que estava aguardando o lançamento por algum motivo.

Se alguém tiver opiniões e puder compartilhar.

Abraços!

**Comment by lddv** [comment]
Meu conhecimento sobre a Stripe no Canadá é que eles tem um filtro de fraude que ajuda a não fazer a venda para perfis potencialmente fraudulentos. Isso ajuda em outras métricas como Chargeback - quando é necessário fazer o estorno no cartão de crédito do usuário - pois com ela você teria menos dessas ocorrências.

Não conheço a atuação da Stripe no Brasil e não sei se eles já vão pro mercado brasileiro já munidos desses perfis, ou se estão estimando com base externa. Entendo que para terem sucesso teriam que estudar bastante as técnicas de fraude aplicadas no Brasil.

@wilsonneto-dev talvez uma Prova de Conceito ou um fluxo separado para poucos usuários possa te dar mais dados para decidir.
